Compilation fails linking if there are different versions of libusb

This issue comes from an attempted install of `probe-run-rp` in which cargo would enter linking stage and then fail with various messages of `undefined reference to libusb_xxxx`. The weird part is that it only affected this crate, other crates using `rusb` compiled without even warnings.

This issue was solved after uninstalling the libraries `libusb-dev` and `libusb-0.1-4` and leaving installed only the library `libusb-1.0-0-dev`.

I don't know if this is a bug or expected behaviour (after all, there were may versions of the same library).

--- a/libusb1-sys/build.rs
+++ b/libusb1-sys/build.rs
@@ -168,7 +168,7 @@ fn make_source() {
     base_config.file(libusb_source.join("libusb/strerror.c"));
     base_config.file(libusb_source.join("libusb/sync.c"));
 
-    base_config.compile("libusb.a");
+    base_config.compile("usb-vendored");
     println!("cargo:version_number={}", VERSION);
 }
 

